espresso/docker-compose.yml

```diff
@@ -627,6 +633,34 @@ services:
       start_period: 240s
     restart: unless-stopped
 
+  attestation-service-zk:
+    image: ghcr.io/espressosystems/attestation-verifier-zk:sha-0e987c3
```

Comment:
Where is this docker image created?

Reply:
the repo is here: https://github.com/EspressoSystems/attestation-verifier-zk